src/components/ArchivePage/WorkshopInfoItem.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Chip, Typography, Link, ListItem } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+	listItem: {
+		display: 'flex',
+		flexDirection: 'column'
+	},
+	links: {
+		display: 'flex',
+		flexDirection: 'row',
+		flexWrap: 'wrap'
+	},
+	chips: {
+		display: 'flex',
+		alignItems: 'center',
+		flexWrap: 'wrap'
+	},
+	chip: {
+		margin: theme.spacing(0.125, 0.25, 0.125, 0)
+	}
+
+}));
+
+function LinkItem({ link, text, divider, ...props }) {
+	return <Typography>
+		{divider ? <span>&#183;&nbsp;</span> : null }
+		<Link href={link} {...props} >{text}&nbsp;</Link>
```

Comment:
use ``{` `}`` instead of `&nbsp`. Just put any special character in quotes so we don't have to look up the encoding.